CHANGELOG.txt

```diff
@@ -100,6 +100,7 @@
                 - The class 'views.ViewsTestCase' is deprecated ;
                   use the new method 'CremeTestCase.add_credentials()' instead.
             - Javascript :
+                * Refactor creme.widget.DateRangeSelector widget.
```

Comment:
You are in the deprecation section => move to the "breaking" section and :
 - list changes
 - or let the old names in comment in the code.